sync/src/types.rs

```diff
@@ -1009,10 +1020,29 @@ impl<CS: ChainStore> SyncSharedState<CS> {
         self.tx_filter.lock().insert(hash);
     }
 
+    pub fn mark_as_known_txes(&self, hashes: Vec<H256>) {
```

Comment:
Please use `txs` as `transaction`'s plural form abbreviation:

```suggestion
    pub fn mark_as_known_txs(&self, hashes: Vec<H256>) {
```

Reply:
thanks.